qiita_db/test/test_util.py

```diff
@@ -616,6 +619,32 @@ def test_check_access_to_analysis_result(self):
 
         self.assertEqual(obs, exp)
 
+    def test_add_message(self):
+        users = [User('shared@foo.bar'), User('admin@foo.bar')]
+        add_message("TEST MESSAGE", users)
+
+        obs = [[x[0], x[1]] for x in User('shared@foo.bar').messages()]
```

Comment:
If someone adds a message in the populate_test_db.sql file the 4 is gonna change, could you replace for a `count` and then `+ 1`? Same for the tests below.